lib/vagrant-libvirt/action/start_domain.rb

```diff
@@ -178,16 +178,21 @@ def call(env)
               end
 
               # TPM
-              if config.tpm_path
+              if config.tpm_path || config.tpm_version
                 raise Errors::FogCreateServerError, 'The TPM Path must be fully qualified' unless config.tpm_path[0].chr == '/'
 
+                if config.tpm_version
+                  raise Errors::FogCreateServerError, 'The TPM Version only works with the `emulator` backend type' unless config.tpm_type == 'emulator'
+                end
+
                 tpm = REXML::XPath.first(xml_descr, '/domain/devices/tpm')
                 if tpm.nil?
```

Comment:
Do you known why `tpm` is always `nil`?

Every-time I start an stopped domain the tpm configuration is not inside `xml_descr`. Looking at the code, I would expect this code path to only be executed when there is a change in the tpm settings. So I'm not sure what is going on. Thou is that in the end it ends up working inside the VM, but something odd is happening here.